config/locales/devise/en.yml

```diff
@@ -150,5 +154,6 @@ en:
         confirm_with_voice: Confirm with voice call
       otp_method:
         title: How would you like to receive your passcode?
+        instruction: You can change your choice at any time
```

Comment:
To me, "any time" implies that there is a setting where they can change their choice, even after authenticating, but we don't have that feature. The only times they can make a choice are:
- right before they are about to receive an OTP, such as on the phone setup screen or on the delivery method screen
- when already having received an OTP and deciding they want to change the delivery method ("call me with the code instead", or "text me the code instead")